bin/ch/ch.cpp

```diff
@@ -196,97 +196,42 @@ HANDLE GetFileHandle(LPCWSTR filename)
     return GetStdHandle(STD_OUTPUT_HANDLE);
 }
 
-HRESULT CreateLibraryByteCodeHeader(LPCSTR contentsRaw, JsFinalizeCallback contentsRawFinalizeCallback, DWORD lengthBytes, LPCWSTR bcFullPath, LPCSTR libraryNameNarrow)
+HRESULT CreateLibraryByteCode(LPCSTR contentsRaw)
 {
-    HANDLE bcFileHandle = nullptr;
     JsValueRef bufferVal;
     BYTE *bcBuffer = nullptr;
     unsigned int bcBufferSize = 0;
-    DWORD written;
-    // For validating the header file against the library file
-    auto outputStr =
-        "//-------------------------------------------------------------------------------------------------------\n"
-        "// Copyright (C) Microsoft. All rights reserved.\n"
-        "// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.\n"
-        "//-------------------------------------------------------------------------------------------------------\n"
-        "#if 0\n";
-
-    std::string normalizedContentStr;
-    char* nextToken = nullptr;
-    char* token = strtok_s((char*)contentsRaw, "\r", &nextToken);
-    while (token)
-    {
-        normalizedContentStr.append(token);
-        token = strtok_s(nullptr, "\r", &nextToken);
-    }
-    // We no longer need contentsRaw, so call the finalizer for it if one was provided
-    if (contentsRawFinalizeCallback != nullptr)
-    {
-        contentsRawFinalizeCallback((void*)contentsRaw);
-    }
-
-    const char* normalizedContent = normalizedContentStr.c_str();
```

Comment:
This was used to put the entirety of the JS into the header file inside an `#if 0` - it was meant for reference BUT IMO was of little use as the JS files are in the same folder, hence scrapped.